c2corg_ui/static/js/map/map.js

```diff
@@ -973,6 +974,20 @@ app.MapController.prototype.simplifyFeature_ = function(feature) {
 /**
  * @export
  */
+app.MapController.prototype.toggleFullscreen = function() {
```

Comment:
Is this function used/usable for the advanced search maps as well?

Reply:
i think not actually, but with little change on css maybe.

We can do that next time.